release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2229,7 +2229,7 @@ sourceStrategy:
 
 * Previously, the Cluster DNS Operator used GO Kubernetes libraries that were based on Kubernetes 1.24 while {product-title} 4.12 is based on Kubernetes 1.25. With this update, GO Kubernetes API is v1.25.2, which aligns the Cluster DNS Operator with {product-title} 4.12 that uses Kubernetes 1.25 APIs. (link: https://issues.redhat.com/browse/OCPBUGS-1558[*OCPBUGS-1558*])
 
-* Previously, setting the `disableNetworkDiagnostics` configuration to `true` did not persist when the `network-operator` pod was re-created. With this update, the `disableNetworkDiagnostics` configuration property of network`operator.openshift.io/cluster` no longer resets to its default value after network operator restart. (link:https://issues.redhat.com/browse/OCPBUGS-392[*OCPBUGS-392*])
+* Previously, setting the `disableNetworkDiagnostics` configuration to `true` did not persist when the `network-operator` pod was re-created. With this update, the `disableNetworkDiagnostics` configuration property of network operator.openshift.io/cluster no longer resets to its default value after network operator restart. (link:https://issues.redhat.com/browse/OCPBUGS-392[*OCPBUGS-392*])
```

Comment:
Hi! Thanks for the PR! Based on the [issue](https://github.com/openshift/openshift-docs/issues/64739) this should be:
```suggestion
* Previously, setting the `disableNetworkDiagnostics` configuration to `true` did not persist when the `network-operator` pod was re-created. With this update, the `disableNetworkDiagnostics` configuration property of network `operator.openshift.io/cluster` no longer resets to its default value after network operator restart. (link:https://issues.redhat.com/browse/OCPBUGS-392[*OCPBUGS-392*])
```
There was just a missing space between network and `operator.openshift.io/cluster`. The backticks should stay. See [this section](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#:~:text=Use-,backticks,-for%20all%20non) of the contributor guide for more information on backticks. 😄

Reply:
okay thanks @opayne1  I'll do the changes ..